sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/TagsOperations.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary> The Tags service client. </summary>
+    public class TagsOperations : OperationsBase
+    {
+        /// <summary>
+        /// The resource type for Tags.
+        /// </summary>
+        public static readonly ResourceType ResourceType = "Microsoft.Resources/tags";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TagsOperations"/> class for mocking.
+        /// </summary>
+        protected TagsOperations()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TagsOperations"/> class.
+        /// </summary>
+        /// <param name="clientContext"></param>
+        /// <param name="subscriptionGuid"> The Guid of the subscription. </param>
+        internal TagsOperations(ClientContext clientContext, string subscriptionGuid)
+            : base(clientContext, new SubscriptionResourceIdentifier(subscriptionGuid))
+        {
+        }
+
+        /// <summary>
+        /// Gets the valid resource type for this operation class.
+        /// </summary>
+        protected override ResourceType ValidResourceType => ResourceType;
+
+        private TagsRestOperations RestClient => new TagsRestOperations(Diagnostics, Pipeline, ((SubscriptionResourceIdentifier)Id).SubscriptionId, BaseUri);
+
+        /// <summary> This operation allows deleting a value from the list of predefined values for an existing predefined tag name. The value being deleted must not be in use as a tag value for the given tag name for any resource. </summary>
+        /// <param name="tagName"> The name of the tag. </param>
+        /// <param name="tagValue"> The value of the tag to delete. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response> DeleteValueAsync(string tagName, string tagValue, CancellationToken cancellationToken = default)
```

Comment:
tagname should be part of the id here if we are treating a tag as a resource.  Unless something else will be its unique id?

Reply:
Done